homeassistant/components/google_assistant/__init__.py

```diff
@@ -29,10 +29,10 @@
     DOMAIN,
     SERVICE_REQUEST_SYNC,
 )
-from .const import EVENT_COMMAND_RECEIVED, EVENT_SYNC_RECEIVED  # noqa: F401
-from .const import EVENT_QUERY_RECEIVED  # noqa: F401
 from .http import GoogleAssistantView, GoogleConfig
```

Comment:
Pretty sure this needs to stay here for the tests to pass. 
I would not recommend sorting this file at all with isort since it removes groups these important imports.

Reply:
I will fix this later today.

For https://github.com/home-assistant/home-assistant/pull/29739 to get merged, the issues here should be fixed!

Reply:
Great!